controllers/brigade/postBrigade.js

```diff
@@ -131,6 +131,8 @@ function postBrigade (req, res, next) {
       thisBrigade.auth.segment.writeKey = req.body['segment-write-key']
       thisBrigade.auth.email.user = req.body['emailuser']
       thisBrigade.auth.email.password = req.body['emailpass']
+    } else if (req.body.cssOverride) { // css override updated
+      thisBrigade.cssOverride = req.body.cssOverride
```

Comment:
As a general comment, do we want to start breaking these out into separate controllers instead of going off the body of the request?  This just seems messy.